client/landing/gutenboarding/onboarding-block/style-preview/style.scss

```diff
@@ -2,12 +2,12 @@
 	display: grid;
 	grid-template-areas: 'title viewport-select actions';
 	column-gap: 1em;
-	margin-top: 53px;
+	margin-top: 60px;
 }
 
 .style-preview__content {
 	display: grid;
-	margin-top: 55px;
+	margin-top: 64px;
```

Comment:
This value (64px) looks like something we'd like to keep in a variable. Something like `$gutenboarding-heading-margin` would work.

Reply:
Done